src/pages/About/Staff.vue

```diff
@@ -24,6 +24,11 @@
           :double-set="false"
         />
       </div>
+      <OfficerCardSet
```

Comment:
Add a break <br /> and a div with the same name as the div for the first officerCardSet